.github/workflows/avm.res.network.service-endpoint-policy.yml

```diff
@@ -0,0 +1,85 @@
+# >>> UPDATE to for example "avm.res.azure.key-vault.vault"
```

Comment:
Please remove comments like the above :)

Reply:
Done.